document/RuleMetadataProfile.mdk

```diff
@@ -2,13 +2,13 @@
 
 ## General
 
-It is sometimes useful for an issue management system to provide information to the end user
-about the rules that were violated by the issues in a log file.
-For example, the user might want to know whether the issue is a potential security risk,
-or where to find additional documentation for the issue.
+It is sometimes useful for a result management system to provide information to the end user
+about the rules that were evaluated in the course of producing the results in a log file.
+For example, the user might want to know whether the result indicates a potential security risk,
```

Comment:
@michaelcfanning I'm going to stick with "result" here, for two reasons:

1. For parallelism with the previous sentence ("... the results in a log file").
2. Because what the user sees in the log file are "results" (some of which, based on their `kind`, are understood to be "problems").